vipps-login-benefits.md

```diff
@@ -0,0 +1,47 @@
+### Benefits in Vipps through Vipps Login
+
+Note: *This is currently work in progress and not part of the official documentation for Vipps Login*
+
+Vipps Login offers the merchant to publish benefits to the end user in the Vipps App. A benefit could be a discount coupon that the end eser will see in the Vipps app after signing up to your customer club or loyalty scheme by using Vipps Login
+
+## Consent to receiving benefits
+In order for Vipps to be able to expose benefits from a merchant, we need to gather an explicit consent from the end user. This consent is obtained in the Vipps Login in Browser flow. The user will be presented with the consent to benfits scrreen if the OIDC scope `promotions` is requested as part of the initial `/auth` request. When the `promotions` scope is requested the user will be presented with this screen as part of the login flow: 
```

Comment:
```suggestion
Before Vipps can expose benefits from a merchant, we need to gather an explicit consent from the end user. This consent is obtained in the Vipps Login, Browser flow.  If the OIDC scope `promotions` is requested as part of the initial `/auth` request, the user will be presented with the *consent to benefits* screen.  For example: 
```

I had a little trouble understanding at first, but I think this might help... If I understood correctly.

Reply:
Thanks for the rewrite :-)

Reply:
For some reason I'm unable to commit the suggestions directly in Github - will have to apply this change manually